engines/misc/src/test/java/org/alfresco/transform/misc/transformers/TextToPdfContentTransformerTest.java

```diff
@@ -146,6 +155,78 @@ public void testUTF8WithoutBOM() throws Exception
         transformTextAndCheck("UTF-8", null, false, "31 20 49 20 6d 75 73 74");
     }
 
+    /**
+     * Test if a different font can be chosen to perform the transformation
+     *
+     * @throws Exception
+     */
+    @Test
+    public void testMNT23960_TimesBold_WithoutAbreve() throws Exception
+    {
+        File sourceFile = File.createTempFile("TMP_Times-Bold", ".txt");
+        String encoding = "UTF-8";
+
+        writeToFile(sourceFile, TEXT_WITHOUT_ABREVE, encoding, null, null);
+
+        Map<String, String> parameters = new HashMap<>();
+        parameters.put(PDF_FONT, PDType1Font.TIMES_BOLD.getName());
+        parameters.put(PDF_FONT_SIZE, "30");
+
+        TransformCheckResult result = transformTextAndCheck(sourceFile, encoding, TEXT_WITHOUT_ABREVE, String.valueOf(-1), true,
+                parameters, false);
+
+        assertEquals(result.getUsedFont(), PDType1Font.TIMES_BOLD.getName());
+        assertNull(result.getErrorMessage());
+    }
+
+    /**
+     * Test if the default font is used when the chosen one is not found
+     *
+     * @throws Exception
+     */
+    @Test
+    public void testMNT23960_InexistentFont_WithoutAbreve() throws Exception
+    {
+        File sourceFile = File.createTempFile("TMP_MyDummyFont", ".txt");
+        String encoding = "UTF-8";
+
+        writeToFile(sourceFile, TEXT_WITHOUT_ABREVE, encoding, null, null);
+
+        Map<String, String> parameters = new HashMap<>();
+        parameters.put(PDF_FONT, "MyDummyFont");
+
+        TransformCheckResult result = transformTextAndCheck(sourceFile, encoding, TEXT_WITHOUT_ABREVE, String.valueOf(-1), true,
+                parameters, false);
+
+        assertEquals(result.getUsedFont(), PDType1Font.TIMES_ROMAN.getName());
+        assertNull(result.getErrorMessage());
+    }
+
+    /**
+     * Test if a different font can be chosen to perform the transformation with abreve character. This test
```

Comment:
I think it's "breve" not "abreve": https://en.wikipedia.org/wiki/Breve
```suggestion
     * Test if a different font can be chosen to perform the transformation with breve character. This test
```

Reply:
As the error fixed by this PR mentioned the `abreve` word I tended to use it across the code, however, I have changed the `abreve` occurrences to `breve`.

> java.lang.IllegalArgumentException: U+0103 ('abreve')